sklearn/multiclass.py

```diff
@@ -637,3 +649,481 @@ def predict(self, X):
         Y = np.array([_predict_binary(e, X) for e in self.estimators_]).T
         pred = euclidean_distances(Y, self.code_book_).argmin(axis=1)
         return self.classes_[pred]
+
+
+def _binomialCoeff(n, k):
```

Comment:
@hlin117 indeed, i adapted it to use this function
@GaelVaroquaux I ran pep8 (and pyflakes) on my changes and saw nothing about that line. Anyway, I removed that method and checked again all the other methods/functions/classes about caps/underscores and it seems it was the only function with that kind of naming.

Thanks for the comments.